PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1695,6 +1775,26 @@ struct HfTrackIndexSkimCreator {
     }
   }
 
+  /// Method to perform ML selections for 2-prong candidates after the rectangular selections
+  /// \param featuresCand is the vector with the candidate features
+  /// \param outputScores is the array of vectors with the output scores to be filled
+  /// \param isSelected ia s bitmap with selection outcome
+  void is3ProngSelectedMlForHfFilters(std::vector<float> featuresCand, std::array<std::vector<float>, kN3ProngDecays>& outputScores, int& isSelected)
```

Comment:
```suggestion
  void applyMlSelectionForHfFilters3Prong(std::vector<float> featuresCand, std::array<std::vector<float>, kN3ProngDecays>& outputScores, int& isSelected)
```

Reply:
Done in https://github.com/AliceO2Group/O2Physics/pull/5430/commits/87b013b121005169bdec56960b9f53ee73e735c4